CHANGELOG.md

```diff
@@ -169,6 +169,8 @@
 - PR #6854 Fix the parameter order of writeParquetBufferBegin
 - PR #6855 Fix `.str.replace_with_backrefs` docs examples
 
+- PR #6853 Fix contiguous split of null string columns
```

Comment:
Looks like you still have a blank line between 6855 and 6853.

Reply:
Sorry, I missed it before. Now, it's removed.